scripts/build/download-tool.sh

```diff
@@ -78,7 +78,7 @@ if [ -f "$ARGV_OUTPUT" ]; then
   fi
 fi
 
-wget --no-check-certificate "$ARGV_URL" -O "$TEMP_OUTPUT"
+wget --no-verbose --no-check-certificate "$ARGV_URL" -O "$TEMP_OUTPUT" 2>&1
```

Comment:
Sounds good. Most CI servers define a `CI` env var that we can use to check it. Regarding the `stderr` piping, it was part of my PowerShell debugging attempts, and it looks like I forgot to remove it.